src/providers/google/chatComplete.ts

```diff
@@ -64,7 +67,17 @@ export const GoogleChatCompleteConfig: ProviderConfig = {
                         }
                     });
                 }
-                return { role, parts };
+
+                // @NOTE: This takes care of the "Please ensure that multiturn requests alternate between user and model."
+                // error that occurs when we have multiple user messages in a row.
+                const shouldAppendEmptyModeChat = lastRole === 'user' && role === 'user' && !params.model?.includes('vision');
```

Comment:
Opting out in case of `gemin-pro-vision` because it doesn't support multiturn chat anyway.